.ci/tav.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
This is the configuration file with the default TAV/Node.js matrix support to run:
- by default if GitHub command
- when a merge commit in main